ouroboros-consensus-protocol/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 # Changelog entries
 
+<a id='changelog-0.5.0.7'></a>
+## 0.5.0.7 — 2023-09-27
+
+### Non-Breaking
```

Comment:
Isn't this a `Patch` change, not `Non-Breaking`?

Reply:
Ah thanks, this should indeed be `Patch`, will change :+1: